components/Calendar/index.js

```diff
@@ -1,20 +1,79 @@
 import Calendar from "react-calendar";
 import "react-calendar/dist/Calendar.css";
 import { useState } from "react";
-// implement styles.js later
+import useCalorieStore from "../../utils/useCalorieStore";
+import styled from "styled-components";
 
 export default function HomeCalendar() {
   const [date, setDate] = useState(new Date());
+  const { history } = useCalorieStore();
+
+  function getTileClassName(date) {
+    if (
+      history.find(
+        (entry) => entry.date === date.getTime() && entry.entry.wasExceeded
+      )
+    ) {
+      return "react-calendar__tile--wasExceeded";
+    } else if (
+      history.find(
+        (entry) => entry.date === date.getTime() && !entry.entry.wasExceeded
+      )
+    ) {
+      return "react-calendar__tile--wasNotExceeded";
+    }
+  }
 
   return (
-    <>
+    <CalendarWrapper>
       <Calendar
+        value={date}
+        selectRange={false}
+        maxDate={new Date(2025, 4, 4)} // replace with today + X later
+        minDate={new Date(2023, 3, 1)} // replace with earliest stored date later
+        minDetail="month"
+        formatShortWeekday={formatShortWeekday}
+        formatMonthYear={formatMonthYear}
+        // ---
         onChange={(date) => {
           setDate(date);
           console.log(date);
         }}
-        value={date}
+        // ---
+        className="calendar-style"
```

Comment:
Where is this class defined, inside of the `react-calendar` library?,

Reply:
In the CalendarWrapper styled component below